### ISSUE TITLE
Audio controls (play button) within the 'Anki export' modal

### ISSUE THREAD

**Issue by Azv121** [killergerbah/asbplayer#560]
**Is your feature request related to a problem? Please describe.**
The subtitles aren't always lined up correctly with the audio. When exporting the audio words are cut off or the next subtitle line is included. I would then have to go back into the export window and play around with the scrubber to capture the exact audio segment to match the subtitles. Even then it's all guess work to capture the correct audio segment.

**Describe the solution you'd like**
Audio controls within the 'Anki export' modal. If there was a play button included near the scrubber at the bottom of the modal I can capture the exact audio which matches the subtitles without having to guess before exporting to Anki.

**Additional context**
![playAnkiExport](https://github.com/user-attachments/assets/83d671f9-6c5c-4545-b973-f69a697bf548)

**Comment by killergerbah** [comment]
@Azv121 Just want to confirm - you are using the asbplayer website with local video?

**Comment by Azv121** [comment]
Yes, I'm using the asbplayer website with local video stored on my drive.

**Comment by killergerbah** [comment]
@Azv121 i guess the UI could be improved but you can already play the audio by clicking on the audio field of the popup

**Comment by Azv121** [comment]
I wasn't aware of this, saves me a lot more time. Thanks for letting me know. :)
